src/components/pill/test/e2e/pill.spec.js

```diff
@@ -0,0 +1,86 @@
+import { assert, getDriver, config } from '../../../../utils/test';
+import { VlPill, VlButtonPill } from './pill.js';
+
+const { sbUrl } = config;
+const defaultUrl = `${sbUrl}?id=custom-elements-vl-pill--default`;
+const successUrl = `${sbUrl}?id=custom-elements-vl-pill--default&args=type:success`;
```

Comment:
Waarom herbruik je je `defaultUrl` niet? Ik heb dat op deze manier gedaan; https://github.com/milieuinfo/uig-webcomponents/blob/feature/tests/src/components/action-group/test/e2e/action-group.spec.js

Op voorhand definiëren is zeker even goed, maar dan zou ik de `defaultUrl` wel hergebruiken.

Reply:
Fixed.